File: AniDAAPI/AniDAAPI/Models/ArrivalInfo.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace AniDAAPI.Models
{
    public partial class ArrivalInfo
    {
        public int ArrInfoId { get; set; }
        public int CustQuoteMasterId { get; set; }
        public string VslCallCondtn { get; set; }
        public decimal? Hforob { get; set; }
        public decimal? Mdorob { get; set; }
        public decimal? LubRob { get; set; }
        public decimal? GasOilRob { get; set; }
        public decimal? Fwrob { get; set; }
        public bool? MsgSent { get; set; }
        public int? LastUpdatedBy { get; set; }
        public DateTime? LastUpdatedDt { get; set; }
    }
}

File: DA_CustomerWebPortal/lib/Model/ArrivalInfo.dart
class ArrivalInfo {
  int arrInfoID;
  int custQuoteMasterID;
  String vslCallCondtn;
  double hforob;
  double mdorob;
  double lubROB;
  double gasOilROB;
  double fwrob;
  //bool msgSent;
  //int lastUpdatedBy;
  //String lastUpdatedDT;

  ArrivalInfo({
    this.arrInfoID,
    this.custQuoteMasterID,
    this.vslCallCondtn,
    this.hforob,
    this.mdorob,
    this.lubROB,
    this.gasOilROB,
    this.fwrob,
    //this.msgSent,
    //this.lastUpdatedBy,
    //this.lastUpdatedDT
  });

  factory ArrivalInfo.fromJson(Map<String, dynamic> json) {
    return ArrivalInfo(
      arrInfoID: json['arrInfoId'],
      custQuoteMasterID: json['custQuoteMasterId'],
      vslCallCondtn: json['vslCallCondtn'],
      hforob: json['hforob'],
      mdorob: json['mdorob'],
      lubROB: json['lubRob'],
      gasOilROB: json['gasOilRob'],
      fwrob: json['fwrob'],
      //msgSent: json['msgSent'],
      //lastUpdatedBy: json['lastUpdatedBy'],
      //lastUpdatedDT: json['lastUpdatedDT'],
    );
  }
}
